### ISSUE TITLE
Error when creating workstation only

### ISSUE THREAD

**Issue by dulao5** [luyomo/OhMyTiUP#2]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?
creating workstation only
```
workstation:
  cidr: 172.82.0.0/16
  instance_type: c5.2xlarge
  keyname: test_key
  keyfile: /home/fuga/.ssh/hoge_test_key.pem
  username: ec2-user
  imageid: ami-098e42ae54c764c35
  volumeSize: 300
```

`bin/aws tidb deploy test embed/examples/test/aws.yaml`

But there is the following error :

```
➜  OhMyTiUP git:(main) ✗ bin/aws tidb deploy test-fortest embed/examples/test-dzg/aws.yaml
Please confirm your topology:
AWS Region:
Cluster type:    tidb
Cluster name:    test-fortest
Cluster version:
User Name:
Key Name:

Component    # of nodes  Instance Type  Image Name             CIDR           User
---------    ----------  -------------  ----------             ----           ----
Workstation  1           c5.2xlarge     ami-098e42ae54c764c35  172.82.0.0/16  admin
Attention:
    1. If the topology is not what you expected, check your yaml file.
    2. Please confirm there is no port/directory conflicts in same host.
Do you want to continue? [y/N]: (default=N) y


+ Deploying all the sub components for tidb solution service
+ Deploying all the sub components for tidb solution service
+ Deploying all the sub components for tidb solution service
+ Deploying all the sub components for tidb solution service
+ Deploying all the sub components for tidb solution service
+ Deploying all the sub components for tidb solution service
  - Preparing workstation ... Done
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x104e339d4]

goroutine 1 [running]:
github.com/luyomo/tisample/pkg/aws/task.newParallelStepDisplay({0x104f199b3, 0x29}, 0x0, {0x14000b0f580, 0x1, 0x1400047f6f0?})
	/Users/fuga/src/pingcap/OhMyTiUP/pkg/aws/task/step.go:149 +0x194
github.com/luyomo/tisample/pkg/aws/task.(*Builder).ParallelStep(...)
	/Users/fuga/src/pingcap/OhMyTiUP/pkg/aws/task/builder.go:472
github.com/luyomo/tisample/pkg/aws/manager.(*Manager).TiDBDeploy(_, {_, _}, {_, _}, {{_, _}, _, {_, _}, ...}, ...)
	/Users/fuga/src/pingcap/OhMyTiUP/pkg/aws/manager/tidb.go:193 +0x1654
github.com/luyomo/tisample/components/aws/command.newTiDBDeploy.func1(0x1400050f180?, {0x1400008af60?, 0x2, 0x2?})
	/Users/fuga/src/pingcap/OhMyTiUP/components/aws/command/tidb.go:70 +0x150
github.com/spf13/cobra.(*Command).execute(0x1400050f180, {0x1400008af00, 0x2, 0x2})
	/Users/fuga/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x4c4
github.com/spf13/cobra.(*Command).ExecuteC(0x1400050e280)
	/Users/fuga/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x34c
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/fuga/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/luyomo/tisample/components/aws/command.Execute()
	/Users/fuga/src/pingcap/OhMyTiUP/components/aws/command/root.go:303 +0x684
main.main()
	/Users/fuga/src/pingcap/OhMyTiUP/components/aws/main.go:23 +0x6c

```

2. What did you expect to see?

success

3. What did you see instead?

4. What version of TiUP are you using (`tiup --version`)?

**Comment by luyomo** [comment]
Resolved by the commit 3a9843a247e67dd7813d68210225aa0f9a3610db

**Comment by luyomo** [comment]
Resolved with commit 3a9843a247e67dd7813d68210225aa0f9a3610db
